stubs/influxdb-client/influxdb_client/client/util/date_utils.pyi

```diff
@@ -0,0 +1,14 @@
+from datetime import datetime, timedelta, tzinfo
+from threading import Lock
+
+date_helper: DateHelper | None
+lock_: Lock
+
+class DateHelper:
+    timezone: tzinfo
+    def __init__(self, timezone: tzinfo = ...) -> None: ...
+    def parse_date(self, date_string: str) -> datetime: ...
```

Comment:
This is a little weird in that the implementation in the class is just `pass`, but `get_date_helper()` monkeypatches the method onto the instances it returns. Annotating it as `-> datetime` seems like the right call but a comment would be helpful.